packages/amplify-codegen/src/utils/getFrontEndFramework.js

```diff
@@ -0,0 +1,13 @@
+const getFrontendHandler = require('./getFrontEndHandler');
+
+function getFrontEndFramework(context) {
```

Comment:
Shouldn't it just be getFrontendJavascript framework? Seems like very specific to javascript.

Reply:
I don't think this code really belongs here. Everything should move to frontend handler eventually. But in the interest of time I am keeping it here